test/Interop/Cxx/stdlib/use-std-vector.swift

```diff
@@ -19,18 +19,17 @@ StdVectorTestSuite.test("init") {
 
 StdVectorTestSuite.test("push back") {
     var v = Vector()
-    var _42: CInt = 42
-    v.push_back(&_42)
+    let _42: CInt = 42
+    v.push_back(_42)
```

Comment:
can we just inline this literal as we do below?

Reply:
Yeah I can do that in a followup. Also need to disable this for template functions too in a follow up.